ZenPacks/zenoss/Microsoft/Windows/migrate/ResetOSProcesses.py

```diff
@@ -0,0 +1,78 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+"""Reset meta_type and portal_type for objects that need it."""
+
+# Logging
+import logging
+LOG = logging.getLogger("zen.Microsoft.Windows.migrate.{}".format(__name__))
+
+# Zenoss Imports
+from zope.event import notify
+from Products.ZenModel.migrate.Migrate import Version
+from Products.ZenModel.ZenPack import ZenPackMigration
+from Products.Zuul.catalog.events import IndexingEvent
+from Products.Zuul.interfaces import ICatalogTool
+
+# ZenPack Imports
+from ZenPacks.zenoss.Microsoft.Windows import progresslog
+
+
+# If the migration takes longer than this interval, a running progress
+# showing elapsed and estimated remaining time will be logged this
+# often. The value is in seconds.
+PROGRESS_LOG_INTERVAL = 10
+
+
+class ResetOSProcesses(ZenPackMigration):
+    version = Version(2, 7, 0)
+
+    def migrate(self, pack):
+        LOG.info("searching for objects")
+        results = ICatalogTool(pack.getDmdRoot("Devices")).search(types=(
+            'ZenPacks.zenoss.Microsoft.Windows.OSProcess.OSProcess',
+            ))
+
+        if not results.total:
+            LOG.info("no objects to migrate")
+            return
+
+        LOG.info("starting: %s total objects", results.total)
```

Comment:
I think making this message more descriptive would be a better way to go instead of the two infos above.  Also, you should use `.format` instead.

`"Found {} Processes that may require indexing".format(results.total)`